aspnetcore/signalr/hubs.md

```diff
@@ -5,7 +5,7 @@ description: Learn how to use hubs in ASP.NET Core SignalR.
 monikerRange: '>= aspnetcore-2.1'
 ms.author: bradyg
 ms.custom: mvc
-ms.date: 11/12/2019
+ms.date: 01/16/2020
```

Comment:
ms.date: 1/16/2020
```suggestion
ms.date: 01/16/2020
```

Reply:
Two problems with suggestions:

* changed don't default to suggested change. Unlike every other PR, I need to manually state this is a suggestion.
* When I do mark as suggesting, it's disabled.